jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -1194,6 +1194,16 @@ ecma_op_object_put_apply_receiver (ecma_value_t receiver, /**< receiver */
     return result;
   }
 
+#if ENABLED (JERRY_ES2015_BUILTIN_PROXY)
```

Comment:
The comment says: `[[Set]] ( P, V, Receiver) operation part for ordinary objects` Proxy is not ordinary, isn't it?

Reply:
I think the comment is still valid. The `ordinary object` concern for the argument `O` not the `Receiver`, which means the operation can be called in a normal object, with a Proxy receiver.